lib/iris/aux_factory.py

```diff
@@ -444,9 +420,8 @@ def calc_bounds():
                                   'These are being disregarded.',
                                   UserWarning, stacklevel=2)
                     orography_pts = nd_points_by_key['orography']
-                    orography_pts_shape = list(orography_pts.shape)
-                    orography = orography_pts.reshape(
-                        orography_pts_shape.append(1))
+                    bds_shape = list(orography_pts.shape) + [1]
+                    orography = orography_pts.reshape(bds_shape)
```

Comment:
Personally I never liked all these called-only-once functions at all.
So given your prompt I've just removed them all -- hope that suits!